### DISCUSSION TITLE
NBT String to NBTComponent

### DISCUSSION THREAD

**Discussion by IceBreathNetDevelopment** [Minestom/Minestom#997] [Help]
I want to get an NBTComponent Object by a NBT String for example: " {BlockEntityTag:{ Patterns:[ { Color:4, Pattern:"tt" },{ Color:5, Pattern:"bts" },{ Color:3, Pattern:"mc" }] } }"

**Comment by jglrxavpok** [comment]
See SNBTReader.

Here's an example:
https://github.com/jglrxavpok/Hephaistos/blob/1e3974ee05c4a47124298cdcbe27b4b83f55458a/common/src/test/java/snbt/SNBTParserTests.java#L156-L164

**Reply by IceBreathNetDevelopment** [reply]
Okay and that is the way how i can transfare "{BlockEntityTag:{ Patterns:[ { Color:4, Pattern:"tt" },{ Color:5, Pattern:"bts" },{ Color:3, Pattern:"mc" }] } }" to a NBTComponent object ?
